lib/ansible/modules/windows/win_regedit.py

```diff
@@ -29,55 +29,69 @@
 DOCUMENTATION = r'''
 ---
 module: win_regedit
-version_added: "2.0"
+version_added: '2.0'
 short_description: Add, change, or remove registry keys and values
 description:
-    - Add, modify or remove registry keys and values.
-    - More information about the windows registry from Wikipedia (https://en.wikipedia.org/wiki/Windows_Registry).
+- Add, modify or remove registry keys and values.
+- More information about the windows registry from Wikipedia
+  U(https://en.wikipedia.org/wiki/Windows_Registry).
 options:
   path:
     description:
-      - Name of registry path.
-      - 'Should be in one of the following registry hives: HKCC, HKCR, HKCU, HKLM, HKU.'
+    - Name of the registry path.
+    - 'Should be in one of the following registry hives: HKCC, HKCR, HKCU,
+      HKLM, HKU.'
     required: true
     aliases: [ key ]
   name:
     description:
-      - Name of registry entry in C(path).
-      - This is an entry in the above C(key) parameter.
-      - If not provided, or empty we use the default name '(default)'.
+    - Name of the registry entry in the above C(path) parameters.
+    - If not provided, or empty then the '(Default)' property for the key will
+      be used.
     aliases: [ entry ]
   data:
     description:
-      - Value of the registry entry C(name) in C(path).
-      - Binary data should be expressed a yaml byte array or as comma separated hex values.  An easy way to generate this is to run C(regedit.exe) and
-        use the I(Export) option to save the registry values to a file.  In the exported file binary values will look like C(hex:be,ef,be,ef).
-        The C(hex:) prefix is optional.
+    - Value of the registry entry C(name) in C(path).
+    - If not specified then the value for the property will be null for the
+      corresponding C(type).
+    - Binary and None data should be expressed in a yaml byte array or as comma
+      separated hex values.
+    - An easy way to generate this is to run C(regedit.exe) and use the
+      I(export) option to save the registry values to a file.
+    - In the exported file, binary value will look like C(hex:be,ef,be,ef), the
+      C(hex:) prefix is optional.
+    - DWORD and QWORD values should either be represented as a decimal number
+      or a hex value.
+    - Multistring values should be passed in as a list.
+    - See the examples for more details on how to format this data.
   type:
     description:
-      - Registry value data type.
-    choices:
-      - binary
-      - dword
-      - expandstring
-      - multistring
-      - string
-      - qword
+    - The registry value data type.
+    choices: [ binary, dword, expandstring, multistring, string, qword ]
     default: string
     aliases: [ datatype ]
   state:
     description:
-      - State of registry entry.
-    choices:
-      - present
-      - absent
+    - The state of the registry entry.
+    choices: [ present, absent ]
     default: present
+  delete_key:
```

Comment:
I would have preferred a more explicit removal of `name=(default)` for the removal of the default key. This is to me more confusing (because whether it is set to `yes` or `no` it will delete a key. So I would get rid of this option altogether.